packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -5,11 +5,141 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../globals/imports';
-
 @mixin leadspace {
   .#{$prefix}--leadspace {
-    background-color: transparent;
+    position: relative;
+    padding-top: carbon--rem(370px) / carbon--mini-units(40) * 100%;
+    height: 0;
+    background: $ui-background;
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__container {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    &__overlay {
+      @include carbon--make-container;
+
+      padding-top: $layout-03;
+      padding-bottom: $layout-03;
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      max-width: none;
+      background-image: linear-gradient(
+        to bottom,
+        rgba($ui-background, 1),
+        transparent
+      );
+    }
+
+    &__content {
+      position: absolute;
+      bottom: $layout-03;
+    }
+
+    .#{$prefix}--leadspace__title {
+      @include carbon--type-style(display-01, true);
+
+      color: $text-01;
+    }
+
+    .#{$prefix}--leadspace__desc {
+      @include carbon--type-style(expressive-heading-03, true);
+
+      color: $text-01;
+    }
+
+    &__ctas {
+      display: flex;
+      flex-direction: column-reverse;
+      margin-top: $layout-01;
+    }
+
+    .#{$prefix}--btn {
+      width: 100%;
+      margin-top: $layout-01;
+    }
+  }
+
+  .#{$prefix}--leadspace--productive .#{$prefix}--leadspace__title {
+    @include carbon--type-style(expressive-heading-06, true);
+  }
+
+  @include carbon--breakpoint(md) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(50) / carbon--mini-units(84) * 100%;
+
+      &__overlay {
+        background-image: linear-gradient(
+          to right,
+          rgba($ui-background, 1),
+          transparent 50%
+        );
+      }
+
+      &__content {
+        width: 100%;
+      }
+
+      .#{$prefix}--leadspace__title {
+        @include carbon--make-row;
```

Comment:
I think this is causing a slight issue with the grid. With an 8/16 col you should be able to get two of these in a row, and it also wraps. I'm not sure if the wrapping is related, or a separate issue at the moment.

![image](https://user-images.githubusercontent.com/3793636/65164899-5b1e9a00-da03-11e9-8c24-c2fb70d2e33e.png)

Reply:
Is the issue here that I should move the @include carbon--make-row into a parent div?

Reply:
currently seeing this 
<img width="1029" alt="Screen Shot 2019-09-19 at 10 23 28 AM" src="https://user-images.githubusercontent.com/54281166/65253009-e6f9fa00-dac7-11e9-8c87-fbeeb14726d7.png">